src/test/java/stepsDesign/GCSBasicDemo.java

```diff
@@ -134,13 +137,18 @@ public void waitTillPipelineIsInRunningState() throws InterruptedException {
     @Then("Verify the pipeline status is {string}")
     public void verifyThePipelineStatusIs(String status) {
         boolean webelement=false;
-        webelement =SeleniumHelper.verifyElementPresent("//*[@data-cy='"+status+"']");
+         webelement=SeleniumHelper.verifyElementPresent("//*[@data-cy='" + status + "']");
         Assert.assertTrue(webelement);
+         log.debug("********Test is Passed************");
+         BeforeActions.scenario.write("********Test is Passed************");
+
     }
 
     @Then("Open Logs")
-    public void openLogs() {
+    public void openLogs() throws InterruptedException {
         CdfPipelineRunAction.logsClick();
+        BeforeActions.scenario.write(CdfPipelineRunAction.captureRawLogs());
+        log.info(CdfPipelineRunAction.captureRawLogs());
```

Comment:
Please write it to a separate file with a reasonable name (e.g. each scenario has it's own file). Don't write through logs. Also location should be configurable with some kind of system property